src/test/java/org/elasticsearch/transport/ContextAndHeaderTransportTests.java

```diff
@@ -122,7 +122,7 @@ public void testThatTermsLookupGetRequestContainsContextAndHeaders() throws Exce
                 .setSource(jsonBuilder().startObject().field("username", "foo").endObject()).get();
         transportClient().admin().indices().prepareRefresh(queryIndex, lookupIndex).get();
 
-        TermsLookupQueryBuilder termsLookupFilterBuilder = QueryBuilders.termsLookupQuery("username").lookupIndex(lookupIndex).lookupType("type").lookupId("1").lookupPath("followers");
+        TermsQueryBuilder termsLookupFilterBuilder = QueryBuilders.termsLookupQuery("username", lookupIndex, "type", "1", "followers");
```

Comment:
I wonder why all these calls changed, it was not needed right? just your personal preference? Double checking that we are bw compatible.